src/fenced/fenced/main.py

```diff
@@ -65,6 +65,17 @@ def panic(reason):
     subprocess.run(['shutdown', '-p', 'now'], check=False)
 
 
+def ensure_log_dir_exists():
+    """
+    On HA systems, we need to ensure that `LOG_DIR` is
+    created so that logging to `LOG_DIR`/fenced.log works.
+    """
+    if not os.path.exists(LOG_DIR):
+        os.makedirs(LOG_DIR, exist_ok=True)
+
+    return
```

Comment:
makes more sense to do all that within the logging.py of fenced?
And reuse the log file var?

Also I am not sure why the fenced in this repo is diverging so much with the 11.3 in truenas repo

Reply:
I'll check repos and see what the divergence is.